chart/kubeapps/values.yaml

```diff
@@ -1423,7 +1423,16 @@ featureFlags:
   ## @param featureFlags.operators Enable ingress record generation for Kubeapps
   ##
   operators: false
-
+  ## @param featureFlags.apiOnly.enabled Enable ingress for API operations only. Access to "/" will not be possible, so Dashboard will be unusable.
+  ## @param featureFlags.apiOnly.grpc.annotations [object] Specific annotations for the GRPC ingress in API-only mode
+  ## For a full list of possible ingress annotations, please see
+  ## ref: https://github.com/kubernetes/ingress-nginx/blob/main/docs/user-guide/nginx-configuration/annotations.md
+  ##
+  apiOnly:
+    enabled: false
+    grpc:
+      annotations:
+        nginx.ingress.kubernetes.io/backend-protocol: GRPC
```

Comment:
Just wondering why this one is configurable - won't our own clients need this specific annotation, even if others are optional/configurable? I expected this one to be hard-coded into the annotations of the ingress. Ah, or perhaps you're just separating that dependency here (ie. other clients may want other annotations). Related question, why does the default in the README show up as `{}` ?

Reply:
> you're just separating that dependency here

Exactly. There is no trace of Nginx in the templates of ingresses. It is only through annotations that it is customized, hence this value.

Reply:
> why does the default in the README show up as {} ?

No idea, it is generated by the readme generator, so changing it manually is not an option :S